README.md

```diff
@@ -38,6 +38,7 @@ docker build .
 * `NSM_FORWARDER_NETWORK_SERVICE_NAME` - the default service name for forwarder discovering (default: "forwarder")
 * `NSM_OPEN_TELEMETRY_ENDPOINT`        - OpenTelemetry Collector Endpoint (default: "otel-collector.observability.svc.cluster.local:4317")
 * `NSM_METRICS_EXPORT_INTERVAL`        - interval between mertics exports (default: "10s")
+* `NSM_PROFILER_HTTP_PORT`             - Profiling server HTTP port providing data in the format expected by the pprof visualization tool. Profiler is running only on images with the "-pprof" tag suffix (default: "6060")
```

Comment:
This parameter is going to be unavailable for non-profile images.

Reply:
Yes, I mentioned that in the description.